content/api/ignore/index.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "Endpoint: ignore"
+tags: ["API"]
+category: "api"
+menu:
+  api:
+    identifier: ignore
+    weight: 400
+---
+
+## /ignore/list
+
+```http
+GET /api/v1/ignore/list.ext HTTP/1.1
+Host: https://yoursite.vanillaforums.com
+```
+
+Get a user’s ignore list.
+
+[__Authentication__](../#making-api-calls): required
```

Comment:
Don't use "../" here. Remember, I told you to use https://gohugo.io/content-management/cross-references/
If we ever move this page under another page, all of these links will break.
`{{< relref "/api/index.md#making-api-calls" >}}`

If this link shows up again and again, let's make a shortcode for it. I'll help you.

Reply:
Forget about the shortcode, it seems like the relref doesn't play nice with shortcodes.